### ISSUE TITLE
Render checkboxes via reference blocks

### ISSUE THREAD

**Issue by tsnieman** [dendronhq/dendron#227]
**Is your feature request related to a problem? Please describe.**
Reference blocks do not render checkboxes:
![image](https://user-images.githubusercontent.com/595711/94290460-0f080e80-ff0f-11ea-9d24-4ad7a6cbc2da.png)

**Describe the solution you'd like**
- Rendered checkboxes in reference blocks
- Rendered, usable checkboxes in reference blocks
- :point_up: Syntax to distinguish between these two? I can imagine some folk might want to embed but not open up the possibility of accidentally modifying the referenced block. As for which is the default... I suspect _usable_ checkboxes would be what most people expect.

**Describe alternatives you've considered**
- n/a

**Additional context**
- n/a

**Comment by kevinslin** [comment]
I agree, I think usable checkboxes are in keeping with user expectation. Caveat is that this is non-trivial to implement - the actual live checkbox toggle relies on some javascript magic. It might be a while before we get around to implementing "live checkboxes". I might break this into two tasks - rendered checkboxes and life checkboxes. The former we can do in October, the latter might take longer just with everything else in the backlog

**Comment by aleksey-rowan** [comment]
Checkboxes are rendered in the current version of the V2 preview.
